### DISCUSSION TITLE
Any need for Product design contribution?

### DISCUSSION THREAD

**Discussion by Elia-Green** [HiEventsDev/Hi.Events#80] [Q&A]
Hi all :) I would love to contribute and wanted to ask if there is anything needed regarding user research, prototyping, and interface design.
Can share my portfolio if relevant,
Cheers

**Comment by daveearley** [comment]
Hi @Elia-Green, Apologies, I missed this! We're open to any contributions! If there's anything in particular in the UI you think should be improved let me know and we can work from there :-)

**Reply by Elia-Green** [reply]
Great! will reach out :)
